build/azuredevops-runner.yml

```diff
@@ -96,19 +113,11 @@ stages:
               testResultsFiles: "**/pester-unittest-results.xml"
               testRunTitle: UnitTests
 
-          - task: PublishCodeCoverageResults@1
+          - task: PublishCodeCoverageResults@2
```

Comment:
Is v2 ready yet? Last I looked it lacked lots of features like line-level coverage?

Reply:
Oh ok, did not know that
I changed it because I was being told that V1 was deprecated.
It seems to be running ok.

Reply:
https://github.com/microsoft/azure-pipelines-tasks/issues/19669

Reply:
OK sure, let's keep an eye on that issue above. 🙂

Reply:
Reverted